### ISSUE TITLE
Remove duplicate requirement section in english on the Shenzhen page.

### ISSUE THREAD

**Issue by tobie** [w3c/testtwf-website#196]
/cc @cptvitamin

**Comment by rhauck** [comment]
Not sure I understand this one - the requirements are listed only once in english, but the highlighted specs are listed twice. Is that what this issue is referencing?

**Comment by tobie** [comment]
Yes. Someone pointed out the duplication to me during TPAC. Sounds like we might want to list the specs in English only and have the text in both languages. Thoughts?
